integration_tests/pages/prisons/sessionTemplates/addSessionTemplate.ts

```diff
@@ -78,6 +78,10 @@ export default class AddSessionTemplatePage extends Page {
     })
   }
 
+  togglePublicVisibility = (): void => {
```

Comment:
This is probably OK but is potentially brittle because it depends on the start (default) state - and that that is correct. An alternative would be something like `setPublicVisibility = (): void => { cy.get('...').check() }` (and a corresponding `unsetPublicVisitbility / uncheck()` ).

Or, `setPublicVisibility = (visible: boolean): void => { // check() / uncheck() based on 'visible' }`